app/tests/src/actions/transaction-actions.js

```diff
@@ -1,4 +1,4 @@
-import {setupInitialState} from './actions'
+import {setStateFn, setupInitialState} from './actions'
```

Comment:
I'd rename this to setupInitialMockState and setMockState (without the 'Fn' suffix as it doesn't convey anything important) to make it clear to the reader that we are not reusing functions from the actual frontend which is otherwise easy to confuse given the relative import of actions from the "tests" folder

Reply:
Will do :+1: